nss-capricious-finches/nutshell
# Jr social

# Link to Ticket
Closes # Link

# Description of Proposed Changes
Steps to Test:
fetch and checkout the branch
make sure the login page comes up and you are able to navigate from it
check the IDs in landing.js


```
git fetch --all
git checkout JR_Social
```

# Impacted Areas in Application
List general components of the application that this PR will affect:
fixed the main page to only have the login function
created IDs for the tiles to call
# Mentions @username
Tag users that need to review this code

# Definition of Done
#### The project must be fully documented. This includes the following:
  1) Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
  2) Every class must be documented with purpose, author, and methods.
  3) Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
  4) The project must be able to run fully, and without errors, on each teammate's system.
  5) Fulfills every requirement. Every line of code has been peer reviewed.
  6) For projects that require unit testing, core functionality must be identified and have at least one test for each.
